### DISCUSSION TITLE
Blank Page when installing on Laravel 9

### DISCUSSION THREAD

**Discussion by syvbyen** [austintoddj/canvas#1297] [Troubleshooting] [answered]
Hey, I just installed Canvas to my Laravel 9 website, and when I tried to log into Canvas I met a basically blank page. 

First I tried to install the newest version. When that didnt work, I thought maybe I should try an earlier version. So I installed the https://github.com/austintoddj/canvas/releases/tag/v6.0.39 version where it added Laravel 9 support.

I also did the:
```
php artisan canvas:install
```

It might be worth to mention that I'm using Laravel Homestead, and sometimes I have some commands that work inside the VM and some that works outside. I don't know if this is related or no. 

I attached some pictures to show what I mean.

![Skjermbilde 2023-06-12 183505](https://github.com/austintoddj/canvas/assets/62508550/59cd253d-a5b5-461a-b4b7-f810e6dac8f8)
![Skjermbilde 2023-06-12 183527](https://github.com/austintoddj/canvas/assets/62508550/76e4cb1a-0128-4da9-9706-43c7c8add2c7)

Here's my composer.json:
```
{
    "name": "laravel/laravel",
    "type": "project",
    "description": "The Laravel Framework.",
    "keywords": ["framework", "laravel"],
    "repositories": [
        {
            "type": "composer",
            "url": "https://repo.backpackforlaravel.com/"
        }
    ],
    "license": "MIT",
    "require": {
        "php": "^8.0.2",
        "austintoddj/canvas": "6.0.37",
        "backpack/crud": "^5.4",
        "backpack/pro": "1.2.8",
        "guzzlehttp/guzzle": "^7.2",
        "laravel/framework": "^9.11",
        "laravel/sanctum": "^2.14.1",
        "laravel/tinker": "^2.7",
        "laravel/ui": "^3.4",
        "spatie/laravel-cookie-consent": "^3.2",
        "spatie/laravel-sluggable": "^3.4"
    },
    "require-dev": {
        "backpack/generators": "^3.1",
        "fakerphp/faker": "^1.9.1",
        "laravel/sail": "^1.0.1",
        "mockery/mockery": "^1.4.4",
        "nunomaduro/collision": "^6.1",
        "phpunit/phpunit": "^9.5.10",
        "spatie/laravel-ignition": "^1.0"
    },
    "autoload": {
        "psr-4": {
            "App\\": "app/",
            "Database\\Factories\\": "database/factories/",
            "Database\\Seeders\\": "database/seeders/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "Tests\\": "tests/"
        }
    },
    "scripts": {
        "post-autoload-dump": [
            "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
            "@php artisan package:discover --ansi"
        ],
        "post-update-cmd": [
            "@php artisan vendor:publish --tag=laravel-assets --ansi --force"
        ],
        "post-root-package-install": [
            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
        ],
        "post-create-project-cmd": [
            "@php artisan key:generate --ansi"
        ]
    },
    "extra": {
        "laravel": {
            "dont-discover": []
        }
    },
    "config": {
        "optimize-autoloader": true,
        "preferred-install": "dist",
        "sort-packages": true
    },
    "minimum-stability": "dev",
    "prefer-stable": true
}
```

**Comment by syvbyen** [comment, marked as answer]
I found the answer. In my config/app.php my fallback_locale was set to "nb" (norwegian bokmål). When I changed this to "en" it worked.

**Reply by austintoddj** [reply]
Apologies on the delay, glad you got it sorted @syvbyen

**Comment by austintoddj** [comment]
Closing this out with [Canvas v7.0.0](https://github.com/austintoddj/canvas/releases/tag/v7.0.0).

v7 requires **PHP 8.3+** and **Laravel 12 or 13**. Laravel 9 is outside the support window. On a supported stack, install with `composer require austintoddj/canvas` and `php artisan canvas:install`, then `canvas:make-admin`.
